spec/support/shared_examples/graphql_request.rb

```diff
@@ -0,0 +1,17 @@
+shared_examples "graphql_request" do |spec_name|
```

Comment:
Would it be readable if we name it with spaces?
```suggestion
shared_examples "graphql request" do |spec_name|
```